pkg/rancher-components/src/components/StringList/StringList.test.ts

```diff
@@ -297,7 +297,7 @@ describe('stringList.vue', () => {
       await inputField.trigger('keydown.enter');
       await wrapper.vm.$nextTick();
 
-      const emitted = (wrapper.emitted('change') || [])[1][0][0];
+      const emitted = (wrapper.emitted('change') || [])[0][0][0];
```

Comment:
When `emits` is specified the value returned by emitted() omits the event type so I had to go through and subtract 1 from each index to account for the lack of event type.

Reply:
This is a good development. This behavior was quite confusing while we were fixing the unit tests and it was unclear that omitting emits options was the root cause.